entrypoints/popup/App.tsx

```diff
@@ -36,16 +36,19 @@ function App() {
         </select>
       </div>
       <div>
-        <label htmlFor="settings-mode">Translated language</label>
+        <label htmlFor="settings-mode">Mode:</label>
         <select id="settings-mode" onChange={handleModeChange}>
           <option selected={mode === TranslateMode.Enabled} value={TranslateMode.Enabled}>
-            Always Show
+            Dual Mode
```

Comment:
I would say this title is more understandable now

Reply:
> I would say this title is more understandable now

Completely agree 👍  Would be nice to update the `TranslateMode` enum as well, but that might be backwards incompatible with previous versions/stored settings. I can follow that up at a later point.